platforms/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = """
 Rust platform registry with information about valid Rust platforms (target
 triple, target_arch, target_os) sourced from Rust Forge.
 """
-version    = "1.1.0" # Also update html_root_url in lib.rs when bumping this
+version    = "2.0.0-pre" # Also update html_root_url in lib.rs when bumping this
```

Comment:
Note: there are already other breaking changes that haven't been released. I'm not just bumping it for this.